spec/unit/action/deploy/module_spec.rb

```diff
@@ -90,8 +90,8 @@
           else
             expect(environment).not_to receive(:generate_types!)
           end
-          @modules << mod
           expect(environment.puppetfile).to receive(:modules).and_return([mod]).twice
+          @modules << mod
```

Comment:
Why did this need to move?

Reply:
:facepalm: It didn't. I was messing with this before block and put it back wrong.